src/main/kotlin/com/atlassian/performance/tools/report/api/judge/Verdict.kt

```diff
@@ -26,6 +28,12 @@ class Verdict(
         logger.info("Performance results are accepted")
     }
 
+    fun failedActions(): List<ActionType<*>> {
+        return reports
+            .filterIsInstance<FailedActionJunitReport>()
```

Comment:
We can add the failed action types to `Verdict` directly. No need to couple them with JUnit reports.
`is instance` sniffing is always a bad idea